www/source/docs/run-packages-upload-files.html.md

```diff
@@ -3,44 +3,42 @@ title: Upload files to a service group
 ---
 
 # Upload files to a service group
-In addition to [configuration updates](/docs/run-packages-apply-config-updates), you can upload files to a service group. The file upload process is very similar to applying configuration updates; however, there is one important difference: You must use [service group encryption](/docs/run-packages-security#service-group-encryption) to upload files to a service group. This requires creating service keys and user keys. File uploads cannot be in the clear.
+In addition to [configuration updates](/docs/run-packages-apply-config-updates), you can upload files to a service group. Keep these small - we recommend 4k or less per file, and keep them to a minimum.
```

Comment:
"Keep them to a minimum" is redundant unless you're trying to say something that "keep these small" isn't already saying.